src/app/shared/services/studio-configuration.ts

```diff
@@ -0,0 +1,21 @@
+import { Configuration } from '../../core/models/configuration'
+
+class mediaPreviewUrl {
+  public mediaPreviewUrl: string
+  constructor(url: string) {
+    this.mediaPreviewUrl = url
+  }
+}
+
+class Settings {
+  public settings: mediaPreviewUrl
+  constructor(url: string) {
+    this.settings = new mediaPreviewUrl(url)
+  }
+}
+export class StudioConfiguration implements Configuration {
+  public data: Settings
+  constructor(url: string) {
+    this.data = new Settings(url)
+  }
+}
```

Comment:
These don't utilize any functionality of being a class, so they might as well just be interfaces.